.github/workflows/publish_docker_images.yml

```diff
@@ -9,101 +9,89 @@ on:
       - main
 
 jobs:
-  docker:
+  push_osdk_image:
```

Comment:
For consistency, let's name this job `publish_osdk_image`. "Publish" means first "build" and then "push".